sdks/go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/apache/beam/sdks
+
+go 1.16
+
+require (
+	cloud.google.com/go v0.88.0 // indirect
+	cloud.google.com/go/bigquery v1.19.0
+	cloud.google.com/go/datastore v1.5.0
+	cloud.google.com/go/pubsub v1.13.0
+	cloud.google.com/go/storage v1.16.0
+	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
+	github.com/golang/protobuf v1.5.2
+	github.com/golang/snappy v0.0.4 // indirect
+	github.com/google/go-cmp v0.5.6
+	github.com/google/uuid v1.3.0
+	github.com/kr/text v0.2.0 // indirect
+	github.com/linkedin/goavro v2.1.0+incompatible
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/nightlyone/lockfile v1.0.0 // indirect
+	github.com/spf13/cobra v1.2.1
+	golang.org/x/net v0.0.0-20210726213435-c6fcb2dbf985
```

Comment:
Yeah, that's a bit strange. It worked on my machine, but that's probably because I have Go 1.17 installed. I'm not sure why `go mod tidy` selected that psuedo-version specifically though. I'm changing it to one that doesn't require Go 1.17 and we'll see how it goes.